packages/lab/src/common-hooks/useCollapsibleGroups.ts

```diff
@@ -67,13 +67,13 @@ export const useCollapsibleGroups = <Item>({
   );
 
   const handleClick = useCallback(
-    (evt) => {
+    (evt: MouseEvent) => {
       console.log(`useCollapsibleGroups idx=${highlightedIdx}`);
       const item = collectionHook.data[highlightedIdx];
       if (
         item &&
         canToggleItem(item) &&
-        (!canSelectItem(item) || toggleIconClicked(evt.target))
+        (!canSelectItem(item) || toggleIconClicked(evt.target as HTMLElement))
```

Comment:
could potentially avoid casting by using `MouseEvent<HTMLElement>` and then passing `evt.currentTarget` instead of `evt.target`

see [typing issue](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/11508#issuecomment-256045682)